src/booleans.js

```diff
@@ -1,62 +1,68 @@
 function negate(a) {
-  // your code here
+  return !a;
+  //
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return !a && !b;
 };
 
 function one(a, b) {
-  // your code here
+  return (a&&!b) || (b&&!a);
 };
 
 function truthiness(a) {
-  // your code here
+  return !!a;
 };
 
 function isEqual(a, b) {
-  // your code here
+  return !!(a==b) ;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  return (a>b);
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  return (a<=b);
 };
 
 function isOdd(a) {
-  // your code here
+  return !((a%2)==0);
```

Comment:
You could simplify this a bit by doing `a % 2 === 0`. It is also standard practice to separate your variables and operators with spaces like in my example, as it is a bit more human friendly to read.

Reply:
You should try to use `===` instead of `==`